src/analyzer/protocol/rawstream/RawStream.cc

```diff
@@ -0,0 +1,25 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#include "zeek/analyzer/protocol/rawstream/RawStream.h"
+
+#include "zeek/analyzer/protocol/rawstream/events.bif.h"
+
+namespace zeek::analyzer::rawstream {
+
+RawStream_Analyzer::RawStream_Analyzer(Connection* conn) : analyzer::tcp::TCP_ApplicationAnalyzer("RawStream", conn) {}
+
+
+void RawStream_Analyzer::DeliverStream(int len, const u_char* data, bool orig) {
+    analyzer::tcp::TCP_ApplicationAnalyzer::DeliverStream(len, data, orig);
+
+    auto s = len > 0 ? zeek::make_intrusive<StringVal>(len, reinterpret_cast<const char*>(data)) :
+                       zeek::val_mgr->EmptyString();
```

Comment:
Just `return` if `len` is zero?

Reply:
I need to admit I don't really no when we'd get `0`, but thought propagating it to script-land seemed reasonable in case it means something, so I'll keep the zero cases.